todomvc/README.md

```diff
@@ -0,0 +1,47 @@
+# TodoMVC
+
+This actor implements the [TodoMVC backend spec](https://github.com/TodoBackend/todo-backend-js-spec/blob/master/js/specs.js).
+
+This actor makes use of the HTTP server (`wasmcloud:httpserver`) capability, the key-value store capability (`wasmcloud:keyvalue`) and the logging capability (`wasmcloud:logging`). As usual, it is worth noting that this actor does _not_ know where its HTTP server comes from, nor does it know which key-value implementation the host runtime has provided.
+
+## How to run
+
+- To start off you will need redis:
+
+```bash
+docker run -p 6379:6379 --name todomvc-store -d redis
+```
+
+- Run `make` and `wash claims inspect target/wasm32-unknown-unknown/debug/todomvc_s.wasm` from which you can extract the `<Actor id>`.
+
+- Run `export TODO_ACTOR=<Actor id (called Module in the above output)>`.
```

Comment:
I have a strong dislike for exporting things into your shell on an ad-hoc basis (the problem Stu had with $KVCACHE_HOST is something that I've had too many times, and prefer to avoid) I think that we should roll this up into a make run .PHONY rule.

- [ ] Really, I feel like the manifest format wants a way to say "whatever id you find at the following path/container registry url", and then you could store the mappings from path to signature in a lockfile that you can optionally check in. I should make a ticket for this.

Reply:
... although I suppose maybe we should leave it like this and see whether the upstream maintainers have an opinion?

Reply:
```sh
wash claims inspect ./target/wasm32-unknown-unknown/debug/todomvc_s.wasm --output json | jq .module -r
```